content/posts/hello-world/index.md

```diff
@@ -32,15 +32,15 @@ We created 2i2c so that these organizations can use entirely open-source tools w
 
 ## Why a non-profit?
```

Comment:
Should this heading also be changed?

Reply:
Yeah I wasn't sure on this one because non-profit seemed generic. If others feel strongly we should change it I'm happy to do so